kubectl-plugin/component/component/auditing.go

```diff
@@ -0,0 +1,27 @@
+package component
+
+import "github.com/kubesphere-sigs/ks/kubectl-plugin/common"
+
+// Auditing return the struct of Auditing
+type Auditing struct {
+}
+
+// GetName return the name of Auditing
+func (e *Auditing) GetName() string {
+	return "auditing"
+}
+
+// Uninstall uninstall Auditing
+func (e *Auditing) Uninstall() error {
+	if err := common.ExecCommand("helm", "uninstall", "kube-auditing", "-n", "kubesphere-monitoring-system"); err != nil {
```

Comment:
Using the helm repository. instead of dependency on binary tools

Reply:
Using the helm repository might be a better solution. Thanks for your suggestion.  We take the command solution for now. @mangoGoForward  or others still have a chance to improve it. How was that sound?

Reply:
Yeah, users will not install helm, we can use helm API.

Reply:
Done.